databases/week3/query.sql

```diff
@@ -0,0 +1,54 @@
+USE `database_week_3`;
+-- Get all Meals
+SELECT * FROM Meal;
+-- Add a new meal
+INSERT INTO Meal (`id`,`title`,`description`,`location`,`when`,`max_reservations`,`price`,`created_date`) VALUES (8,'Rosogolla','Bangladeshi food','ishoj','2023-02-15 07:54:30',10,85.00,'2022-12-23');
```

Comment:
no need to insert `id` manually as the` id` (primary id) is set be auto-incremented (applies where you inserted ids manually)